meteor-app/imports/api/orders/methods.js

```diff
@@ -21,13 +21,25 @@ Meteor.methods({
       throw new Meteor.Error(500, e.sanitizedError.reason)
     }
   },
-  'orders.update'(id, orderedParts) {
+  'orders.update'(orderedPart) {
     try {
-      log.info('updating order: ', id, orderedParts)
-      return Orders.update({ _id: id }, { $push: { orderedParts: { ...orderedParts } } })
+      log.info('updating order: ', orderedPart)
+      return Orders.update({ status: 1 }, { $push: { orderedParts: { ...orderedPart } } })
```

Comment:
I know we talked about this on Friday, but I think the original code was better, where you were passing down the id to the method, telling it exactly which record to update. This new approach will add this part to all orders with status=1, which is really a weak assumption. You might have more than one pending order at a time.
You should also not use the value `1` in your code, you should use the CONSTANTS.ORDER_STATUS.READY (or whatever it is)
It's also good form to name the method to be more descriptive, I would change to be `order.addPart`